iotedgedev/module.py

```diff
@@ -0,0 +1,49 @@
+
+import os
+import json
+
+
+class Module(object):
+    def __init__(self, output, utility, module_json_file):

```

Comment:
I would prefer to pass module_dir into the ctor, then you don't have to pass to each method.